db/seeds.rb

```diff
@@ -1,7 +1,558 @@
+  {name: "Diamond"},
+  {name: "Sapphire"},
+  {name: "Quartz"},
+  {name: "Amethyst"},
+  {name: "Tourmaline"},
+  {name: "Turquoise"},
+]
+
+category_products = [
+  {category_id: "1", product_id: "1"},
+  {category_id: "1", product_id: "2"},
+  {category_id: "1", product_id: "3"},
+  {category_id: "1", product_id: "4"},
+  {category_id: "2", product_id: "5"},
+  {category_id: "2", product_id: "6"},
+  {category_id: "2", product_id: "7"},
+  {category_id: "2", product_id: "8"},
+  {category_id: "2", product_id: "9"},
+  {category_id: "2", product_id: "10"},
+  {category_id: "2", product_id: "11"},
+  {category_id: "2", product_id: "12"},
+  {category_id: "2", product_id: "13"},
+  {category_id: "2", product_id: "14"},
+  {category_id: "2", product_id: "15"},
+  {category_id: "3", product_id: "16"},
+  {category_id: "3", product_id: "17"},
+  {category_id: "3", product_id: "18"},
+  {category_id: "3", product_id: "19"},
+  {category_id: "3", product_id: "20"},
+  {category_id: "11", product_id: "21"},
+  {category_id: "11", product_id: "22"},
+  {category_id: "11", product_id: "23"},
+  {category_id: "11", product_id: "24"},
+  {category_id: "11", product_id: "25"},
+  {category_id: "4", product_id: "26"},
+  {category_id: "4", product_id: "27"},
+  {category_id: "4", product_id: "28"},
+  {category_id: "4", product_id: "29"},
+  {category_id: "4", product_id: "30"},
+  {category_id: "5", product_id: "31"},
+  {category_id: "5", product_id: "32"},
+  {category_id: "5", product_id: "33"},
+  {category_id: "5", product_id: "34"},
+  {category_id: "5", product_id: "35"},
+  {category_id: "5", product_id: "36"},
+  {category_id: "5", product_id: "37"},
+  {category_id: "5", product_id: "38"},
+  {category_id: "6", product_id: "39"},
+  {category_id: "6", product_id: "40"},
+  {category_id: "6", product_id: "41"},
+  {category_id: "6", product_id: "42"},
+  {category_id: "6", product_id: "43"},
+  {category_id: "6", product_id: "44"},
+  {category_id: "7", product_id: "45"},
+  {category_id: "7", product_id: "46"},
+  {category_id: "7", product_id: "47"},
+  {category_id: "7", product_id: "48"},
+  {category_id: "7", product_id: "49"},
+  {category_id: "7", product_id: "50"},
+  {category_id: "8", product_id: "51"},
+  {category_id: "8", product_id: "52"},
+  {category_id: "8", product_id: "53"},
+  {category_id: "8", product_id: "54"}
+]
+
+products.each do |p|
+  prod = Product.new(p)
+  prod.save(validate: false)
+end
+
+categories.each do |c|
+  cat = Category.new(c)
+  cat.save(validate: false)
+end
+
+orderitems.each do |o|
+  orditem = Orderitem.new(o)
+  orditem.save(validate: false)
+end
+
+users.each do |u|
+  user = User.new(u)
+  user.save(validate: false)
+end
+
+orders.each do |o|
+  order = Order.new(o)
+  order.save(validate: false)
+end
+
+reviews.each do |r|
+  review = Review.new(r)
+  review.save(validate: false)
+end
+
+category_products.each do |c|
+  catprod = CategoriesProduct.new(c)
+  catprod.save(validate: false)
+end
```

Comment:
No kidding! Nice looping and use of `validate: false` to insert all them seeds! :seedling: :seedling: :seedling: